packages/web-component/src/i18n/messages.fra.json

```diff
@@ -27,6 +28,10 @@
     "fra": "Français",
     "spa": "Espagnol",
     "close": "Fermer",
-    "annotations-tooltip": "Annotations"
+    "annotations-tooltip": "Annotations",
+    "edit-layer": "Renommer le titre de l'annotation",
+    "delete-layer": "Supprimer le titre de l'annotation",
+    "save-layer": "Enregistrer le titre de l'annotation",
+    "create-layer": "Créer un titre d'annotation"
```

Comment:
And here "la/une couche d'annotation" instead of "le titre".
This is consistent with the translations I did for the tour.